js/roi/ROIManager.js

```diff
@@ -134,9 +134,10 @@ class ROIManager {
         const columns = browser.columnContainer.querySelectorAll('.igv-column')
 
         for (let i = 0; i < columns.length; i++) {
-
+            //span you're looking at
             let {chr, start: viewStart, end: viewEnd, bpPerPixel} = browser.referenceFrameList[i]
 
+            //removing rois not in screen
```

Comment:
again, clean up comments that you added as clarifying notes to yourself before committing